cbins13/dotTheory---FE
# FE-102: Front-End Formations (Christian Binay-an)

## Related Tickets
- [FE-857](https://codeandtheory.atlassian.net/browse/FE-857)

## Description
- This pull request is created for the module FE-102, where the checklist includes three tasks:

1. Add a "Comments" component on your Detail Page
2. Create at least 5 copies of your updated Detail Page, and make sure to change the name and other defining details of the page (e.g. main image, author, etc.)
3. Create the Browse component and link the contents inside it to the different Detail Pages created on the previous task

## Validation
- Open the index.html on Live Server via VSCode to view the detail page, or open the index.html on a browser
- Click on preferred page to navigate to using the header navigation menu found on the top part of the website
- On the index.html page, scroll down to see the Comments component
